views/view_package.dt

```diff
@@ -72,6 +72,23 @@ block body
 				input(id="package-clipboard",value='dub add #{packageName}')
 				button(class="btn-clipboard",data-clipboard-target="#package-clipboard")
 					img(src="/images/clippy.svg", width=13, alt="Copy to clipboard")
+		div
+		.clipboard-manual
+			p
+				strong Manual usage
+				br
+				| Put the following dependency into your project's dependences section:
+			.clipboard-pkg
+				.clipboard-pkg-row
+					.clipboard-pkg-title dub.json
```

Comment:
this is a label, make it a `label` tag and add `(for="package-clipboard-json")` (and adjust CSS to support label instead of div here)

When you click a label the user by default even selects the input, which can be convenient for the user and is more accessible instead of just being some text before a text box.